src/components/Carousel/index.js

```diff
@@ -75,26 +75,63 @@ export default class Carousel extends PureComponent {
     highlightOnHover: false,
     loop: false,
     overflow: false,
-    scrollCount: 1,
+    scrollCount: 2,
     showCount: 3,
     transition: TRANSITION_SLIDE,
   }
 
+  state = {
+    currentSlide: 0,
+  }
+
+  constructor (props) {
+    super(props)
+
+    this.slider = props.sliderRef || React.createRef()
+  }
+
+  handlePrevClick = () => {
+    const { scrollCount } = this.props
+    const { currentSlide } = this.state
+
+    const goTo = currentSlide - scrollCount < 0
```

Comment:
~goTo~ `targetSlide`?  And since we're handling previous click, would it make more sense to do `-1` vs doing the math to subtract `scrollCount` from `currentSlide`? Took me a while to figure this out

Reply:
the value is not the increment, but the actual index to slide to